packages/amplify-category-function/src/provider-utils/awscloudformation/utils/constants.ts

```diff
@@ -9,9 +9,9 @@ export const layerParametersFileName = 'layer-parameters.json';
 export const parametersFileName = 'parameters.json';
 export const provider = 'awscloudformation';
 export const appsyncTableSuffix = '@model(appsync)';
-export const resourceAccessSetting = 'Resource access permissions';
-export const cronJobSetting = 'Scheduled recurring invocation';
-export const lambdaLayerSetting = 'Lambda layers configuration';
+export const resourceAccessSetting = '🔒 Resource access permissions';
+export const cronJobSetting = '⏰ Schedule recurring invocation';
+export const lambdaLayerSetting = '📦 Lambda layers configuration';
```

Comment:
Make sure associated Lambda e2e test helper functions are updated to wait on the updated text.

Reply:
Gotcha! thanks.

Reply:
@renebrandel please ack the icon changes.